src/lib/annotations/doc/DocAnnotator.js

```diff
@@ -390,7 +423,17 @@ class DocAnnotator extends Annotator {
         // Scale existing canvases on re-render
         this.scaleAnnotationCanvases(pageNum);
 
-        super.renderAnnotationsOnPage(pageNum);
+        if (this.threads && this.threads[pageNum]) {
```

Comment:
Actually if you just leave the call to `super.renderAnnotationsOnPage(pageNum)`
We can modify annotator.renderAnnotationsOnPage() to this since w/e annotation types aren't supported will get filtered out w/ the call to `this.isModeAnnotatable(thread.type)`
```
    renderAnnotationsOnPage(pageNum) {
        if (this.threads && this.threads[pageNum]) {
            this.threads[pageNum].forEach((thread) => {
                if (!this.isModeAnnotatable(thread.type)) {
                    return;
                }
                
                thread.show();
            });
        }
    }
```

Plus then it'll be applicable to any other annotation types we don't want to render :)

Reply:
Check if there's anywhere else we can just make calls to `isModeAnnotatable()` to Annotator or AnnotationThread methods to clean up some of hackier logic

Reply:
I'll override the super call to do this, but I still have to not call it, as we're calling our butchered version of `show()` that takes in the flags for plain and comment highlights

Reply:
I'll make a ticket for the cleanup, as I don't want this task to turn into a refactor/cleanup PR. Then we can do it, next